Documentation/getting-started.md

```diff
@@ -68,8 +59,8 @@ will tell you more, check the docs, or feel free to ask in IRC.
 ### 3) Run some storage nodes
 #### Running manually
 ```
-./torusd --etcd 127.0.0.1:2379 --peer-address http://127.0.0.1:40000 --data-dir /tmp/torus1 --size 20GiB
-./torusd --etcd 127.0.0.1:2379 --peer-address http://127.0.0.1:40001 --data-dir /tmp/torus2 --size 20GiB
+torusd --etcd 127.0.0.1:2379 --peer-address http://127.0.0.1:40000 --data-dir /tmp/torus1 --size 20GiB
```

Comment:
It doesn't install them. These docs mix "./torusd" and "torusd" (lots of "torusctl" and "torusblk" commands) but the latter case was more common so I standardized it on that.

Would be happy to switch to prefixing all command with ./bin.